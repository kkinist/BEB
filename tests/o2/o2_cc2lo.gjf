%chk=o2.chk
%nprocs=4
%mem=1000mw
# CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc

BEB step 10: low-spin CCSD(T) for dication of o2

2 1

O 0
cc-pVTZ
****

