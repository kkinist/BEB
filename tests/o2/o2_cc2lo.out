 Entering Gaussian System, Link 0=g09
 Initial command:
 /usr/local/apps/g09/l1.exe "/wrk/irikura/Gau-895.inp" -scrdir="/wrk/irikura/"
 Entering Link 1 = /usr/local/apps/g09/l1.exe PID=       899.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2009,2013,
            Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 09 program.  It is based on
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 09, Revision D.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, B. Mennucci, 
 G. A. Petersson, H. Nakatsuji, M. Caricato, X. Li, H. P. Hratchian, 
 A. F. Izmaylov, J. Bloino, G. Zheng, J. L. Sonnenberg, M. Hada, 
 M. Ehara, K. Toyota, R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, 
 Y. Honda, O. Kitao, H. Nakai, T. Vreven, J. A. Montgomery, Jr., 
 J. E. Peralta, F. Ogliaro, M. Bearpark, J. J. Heyd, E. Brothers, 
 K. N. Kudin, V. N. Staroverov, T. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. Rendell, J. C. Burant, S. S. Iyengar, J. Tomasi, 
 M. Cossi, N. Rega, J. M. Millam, M. Klene, J. E. Knox, J. B. Cross, 
 V. Bakken, C. Adamo, J. Jaramillo, R. Gomperts, R. E. Stratmann, 
 O. Yazyev, A. J. Austin, R. Cammi, C. Pomelli, J. W. Ochterski, 
 R. L. Martin, K. Morokuma, V. G. Zakrzewski, G. A. Voth, 
 P. Salvador, J. J. Dannenberg, S. Dapprich, A. D. Daniels, 
 O. Farkas, J. B. Foresman, J. V. Ortiz, J. Cioslowski, 
 and D. J. Fox, Gaussian, Inc., Wallingford CT, 2013.
 
 ******************************************
 Gaussian 09:  ES64L-G09RevD.01 24-Apr-2013
                17-May-2017 
 ******************************************
 %chk=o2.chk
 %nprocs=4
 Will use up to    4 processors via shared memory.
 %mem=1000mw
 ---------------------------------------------------
 # CCSD(T,maxcyc=100)/GEN gfinput geom=check scf=xqc
 ---------------------------------------------------
 1/29=2,38=1/1;
 2/12=2,40=1/2;
 3/5=7,11=9,14=-4,16=1,24=10,25=1,30=1/1,2,3;
 4//1;
 5/5=2,8=3,13=1,38=5/2,8;
 8/6=4,9=120000,10=1/1,4;
 9/5=7,6=100,14=2/13;
 6/7=2,8=2,9=2,10=2/1;
 99/5=1,9=1/99;
 ------------------------------------------------
 BEB step 10: low-spin CCSD(T) for dication of o2
 ------------------------------------------------
 Structure from the checkpoint file:  "o2.chk"
 Charge =  2 Multiplicity = 1
 Redundant internal coordinates found in file.
 O,0,-2.024167039,1.50180939,0.
 O,0,-3.238759921,1.50180939,0.
 Recover connectivity data from disk.
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          8           0       -2.024167    1.501809    0.000000
      2          8           0       -3.238760    1.501809    0.000000
 ---------------------------------------------------------------------
 Stoichiometry    O2(2+)
 Framework group  D*H[C*(O.O)]
 Deg. of freedom     1
 Full point group                 D*H     NOp   8
 Largest Abelian subgroup         D2H     NOp   8
 Largest concise Abelian subgroup C2      NOp   2
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          8           0        0.000000    0.000000    0.607296
      2          8           0        0.000000    0.000000   -0.607296
 ---------------------------------------------------------------------
 Rotational constants (GHZ):      0.0000000     42.8354962     42.8354962
 General basis read from cards:  (5D, 7F)
 AO basis set in the form of general basis input (Overlap normalization):
      1 0
 S   7 1.00       0.000000000000
      0.1533000000D+05  0.5198089434D-03
      0.2299000000D+04  0.4020256215D-02
      0.5224000000D+03  0.2071282673D-01
      0.1473000000D+03  0.8101055358D-01
      0.4755000000D+02  0.2359629851D+00
      0.1676000000D+02  0.4426534455D+00
      0.6207000000D+01  0.3570644227D+00
 S   6 1.00       0.000000000000
      0.5224000000D+03 -0.4421150084D-04
      0.1473000000D+03 -0.1225910413D-02
      0.4755000000D+02 -0.1055177248D-01
      0.1676000000D+02 -0.6744526326D-01
      0.6207000000D+01 -0.1711986073D+00
      0.6882000000D+00  0.1073298561D+01
 S   1 1.00       0.000000000000
      0.1752000000D+01  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.2384000000D+00  0.1000000000D+01
 P   3 1.00       0.000000000000
      0.3446000000D+02  0.4116348957D-01
      0.7749000000D+01  0.2577628359D+00
      0.2280000000D+01  0.8024192744D+00
 P   1 1.00       0.000000000000
      0.7156000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.2140000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.2314000000D+01  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.6450000000D+00  0.1000000000D+01
 F   1 1.00       0.000000000000
      0.1428000000D+01  0.1000000000D+01
 ****
      2 0
 S   7 1.00       0.000000000000
      0.1533000000D+05  0.5198089434D-03
      0.2299000000D+04  0.4020256215D-02
      0.5224000000D+03  0.2071282673D-01
      0.1473000000D+03  0.8101055358D-01
      0.4755000000D+02  0.2359629851D+00
      0.1676000000D+02  0.4426534455D+00
      0.6207000000D+01  0.3570644227D+00
 S   6 1.00       0.000000000000
      0.5224000000D+03 -0.4421150084D-04
      0.1473000000D+03 -0.1225910413D-02
      0.4755000000D+02 -0.1055177248D-01
      0.1676000000D+02 -0.6744526326D-01
      0.6207000000D+01 -0.1711986073D+00
      0.6882000000D+00  0.1073298561D+01
 S   1 1.00       0.000000000000
      0.1752000000D+01  0.1000000000D+01
 S   1 1.00       0.000000000000
      0.2384000000D+00  0.1000000000D+01
 P   3 1.00       0.000000000000
      0.3446000000D+02  0.4116348957D-01
      0.7749000000D+01  0.2577628359D+00
      0.2280000000D+01  0.8024192744D+00
 P   1 1.00       0.000000000000
      0.7156000000D+00  0.1000000000D+01
 P   1 1.00       0.000000000000
      0.2140000000D+00  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.2314000000D+01  0.1000000000D+01
 D   1 1.00       0.000000000000
      0.6450000000D+00  0.1000000000D+01
 F   1 1.00       0.000000000000
      0.1428000000D+01  0.1000000000D+01
 ****

 There are    16 symmetry adapted cartesian basis functions of AG  symmetry.
 There are     3 symmetry adapted cartesian basis functions of B1G symmetry.
 There are     8 symmetry adapted cartesian basis functions of B2G symmetry.
 There are     8 symmetry adapted cartesian basis functions of B3G symmetry.
 There are     3 symmetry adapted cartesian basis functions of AU  symmetry.
 There are    16 symmetry adapted cartesian basis functions of B1U symmetry.
 There are     8 symmetry adapted cartesian basis functions of B2U symmetry.
 There are     8 symmetry adapted cartesian basis functions of B3U symmetry.
 There are    13 symmetry adapted basis functions of AG  symmetry.
 There are     3 symmetry adapted basis functions of B1G symmetry.
 There are     7 symmetry adapted basis functions of B2G symmetry.
 There are     7 symmetry adapted basis functions of B3G symmetry.
 There are     3 symmetry adapted basis functions of AU  symmetry.
 There are    13 symmetry adapted basis functions of B1U symmetry.
 There are     7 symmetry adapted basis functions of B2U symmetry.
 There are     7 symmetry adapted basis functions of B3U symmetry.
    60 basis functions,   104 primitive gaussians,    70 cartesian basis functions
     7 alpha electrons        7 beta electrons
       nuclear repulsion energy        27.8836981945 Hartrees.
 NAtoms=    2 NActive=    2 NUniq=    1 SFac= 4.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 1 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=    60 RedAO= T EigKep=  6.46D-03  NBF=    13     3     7     7     3    13     7     7
 NBsUse=    60 1.00D-06 EigRej= -1.00D+00 NBFU=    13     3     7     7     3    13     7     7
 ExpMin= 2.14D-01 ExpMax= 1.53D+04 ExpMxC= 5.22D+02 IAcc=1 IRadAn=         1 AccDes= 0.00D+00
 Harris functional with IExCor=  205 and IRadAn=       1 diagonalized for initial guess.
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         1 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=     500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Initial guess orbital symmetries:
       Occupied  (SGU) (SGG) (SGG) (SGU) (SGG) (PIU) (PIU)
       Virtual   (PIG) (PIG) (SGU) (SGU) (PIU) (PIU) (SGG) (PIG)
                 (PIG) (SGG) (DLTG) (DLTG) (SGU) (PIU) (PIU) (DLTU)
                 (DLTU) (SGG) (SGU) (PIG) (PIG) (PIU) (PIU) (SGG)
                 (PIG) (PIG) (DLTG) (DLTG) (PIU) (PIU) (SGU) (SGG)
                 (PHIU) (PHIU) (PHIG) (PHIG) (DLTU) (DLTU) (SGU)
                 (DLTG) (DLTG) (DLTU) (DLTU) (PIG) (PIG) (PIU)
                 (PIU) (SGU) (SGG) (PIG) (PIG) (SGG) (SGU)
 The electronic state of the initial guess is 1-SGG.
 Keep R1 ints in memory in symmetry-blocked form, NReq=3667015.
 Requested convergence on RMS density matrix=1.00D-08 within  64 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RHF) =  -148.206840835     A.U. after    8 cycles
            NFock=  8  Conv=0.73D-08     -V/T= 2.0081
 QCSCF skips out because SCF is already converged.
 ExpMin= 2.14D-01 ExpMax= 1.53D+04 ExpMxC= 5.22D+02 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         5 IDoV=-2 UseB2=F ITyADJ=14
 ICtDFT= 12500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 Range of M.O.s used for correlation:     3    60
 NBasis=    60 NAE=     7 NBE=     7 NFC=     2 NFV=     0
 NROrb=     58 NOA=     5 NOB=     5 NVA=    53 NVB=    53
 Semi-Direct transformation.
 ModeAB=           4 MOrb=             5 LenV=    1048301759
 LASXX=        58091 LTotXX=       58091 LenRXX=      119922
 LTotAB=       61831 MaxLAS=      414120 LenRXY=           0
 NonZer=      178013 LenScr=      785920 LnRSAI=      414120
 LnScr1=     1242112 LExtra=           0 Total=      2562074
 MaxDsk=          -1 SrtSym=           T ITran=            4
 JobTyp=0 Pass  1:  I=   1 to   5.
 (rs|ai) integrals will be sorted in core.
 Spin components of T(2) and E(2):
     alpha-alpha T2 =       0.3680381319D-01 E2=     -0.8581638310D-01
     alpha-beta  T2 =       0.1828898157D+00 E2=     -0.4485059396D+00
     beta-beta   T2 =       0.3680381319D-01 E2=     -0.8581638310D-01
 ANorm=    0.1120935967D+01
 E2 =    -0.6201387058D+00 EUMP2 =    -0.14882697954097D+03
 Keep R2 and R3 ints in memory in symmetry-blocked form, NReq=3623783.
 Iterations= 100 Convergence= 0.100D-06
 Iteration Nr.   1
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        30
 NAB=    15 NAA=     0 NBB=     0.
 MP4(R+Q)=  0.16379664D-01
 E3=        0.16372932D+00        EUMP3=      -0.14866325022D+03
 E4(DQ)=   -0.89298244D-01        UMP4(DQ)=   -0.14875254846D+03
 E4(SDQ)=  -0.11404596D+00        UMP4(SDQ)=  -0.14877729618D+03
 DE(Corr)= -0.43890749     E(Corr)=     -148.64574832    
 NORM(A)=   0.10694441D+01
 Iteration Nr.   2
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        30
 NAB=    15 NAA=     0 NBB=     0.
 DE(Corr)= -0.53366027     E(CORR)=     -148.74050111     Delta=-9.48D-02
 NORM(A)=   0.10870520D+01
 Iteration Nr.   3
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        30
 NAB=    15 NAA=     0 NBB=     0.
 DE(Corr)= -0.52212280     E(CORR)=     -148.72896364     Delta= 1.15D-02
 NORM(A)=   0.10932293D+01
 Iteration Nr.   4
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        30
 NAB=    15 NAA=     0 NBB=     0.
 DE(Corr)= -0.52758968     E(CORR)=     -148.73443051     Delta=-5.47D-03
 NORM(A)=   0.10967850D+01
 Iteration Nr.   5
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        30
 NAB=    15 NAA=     0 NBB=     0.
 DE(Corr)= -0.52835854     E(CORR)=     -148.73519937     Delta=-7.69D-04
 NORM(A)=   0.10972628D+01
 Iteration Nr.   6
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        30
 NAB=    15 NAA=     0 NBB=     0.
 DE(Corr)= -0.52846880     E(CORR)=     -148.73530964     Delta=-1.10D-04
 NORM(A)=   0.10971992D+01
 Iteration Nr.   7
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        30
 NAB=    15 NAA=     0 NBB=     0.
 DE(Corr)= -0.52851778     E(CORR)=     -148.73535861     Delta=-4.90D-05
 NORM(A)=   0.10972490D+01
 Iteration Nr.   8
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        30
 NAB=    15 NAA=     0 NBB=     0.
 DE(Corr)= -0.52848443     E(CORR)=     -148.73532526     Delta= 3.34D-05
 NORM(A)=   0.10972265D+01
 Iteration Nr.   9
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        30
 NAB=    15 NAA=     0 NBB=     0.
 DE(Corr)= -0.52850329     E(CORR)=     -148.73534412     Delta=-1.89D-05
 NORM(A)=   0.10972334D+01
 Iteration Nr.  10
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        30
 NAB=    15 NAA=     0 NBB=     0.
 DE(Corr)= -0.52849759     E(CORR)=     -148.73533843     Delta= 5.70D-06
 NORM(A)=   0.10972334D+01
 Iteration Nr.  11
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        30
 NAB=    15 NAA=     0 NBB=     0.
 DE(Corr)= -0.52849915     E(CORR)=     -148.73533999     Delta=-1.56D-06
 NORM(A)=   0.10972346D+01
 Iteration Nr.  12
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        30
 NAB=    15 NAA=     0 NBB=     0.
 DE(Corr)= -0.52849905     E(CORR)=     -148.73533988     Delta= 1.05D-07
 NORM(A)=   0.10972350D+01
 Iteration Nr.  13
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=        30
 NAB=    15 NAA=     0 NBB=     0.
 DE(Corr)= -0.52849907     E(CORR)=     -148.73533991     Delta=-2.43D-08
 NORM(A)=   0.10972351D+01
 Dominant configurations:
 ***********************
 Spin Case        I    J    A    B          Value
   ABAB           6    6    9    9      -0.188293D+00
   ABAB           7    7    8    8      -0.188293D+00
 Largest amplitude= 1.88D-01
 Time for triples=        2.52 seconds.
 T4(CCSD)= -0.40094691D-01
 T5(CCSD)=  0.35594785D-02
 CCSD(T)= -0.14877187512D+03
 Discarding MO integrals.

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (SGG) (SGU) (SGG) (SGU) (SGG) (PIU) (PIU)
       Virtual   (PIG) (PIG) (SGU) (PIU) (PIU) (SGU) (SGG) (SGG)
                 (PIG) (PIG) (DLTG) (DLTG) (PIU) (PIU) (SGU) (DLTU)
                 (DLTU) (SGG) (SGU) (PIG) (PIG) (PIU) (PIU) (SGG)
                 (PIG) (PIG) (DLTG) (DLTG) (PIU) (PIU) (SGU) (SGG)
                 (PHIU) (PHIU) (PHIG) (PHIG) (DLTU) (DLTU) (SGU)
                 (DLTG) (DLTG) (DLTU) (DLTU) (PIG) (PIG) (PIU)
                 (PIU) (SGU) (SGG) (PIG) (PIG) (SGG) (SGU)
 The electronic state is 1-SGG.
 Alpha  occ. eigenvalues --  -21.90280 -21.90138  -2.65072  -2.04644  -1.70725
 Alpha  occ. eigenvalues --   -1.60966  -1.60966
 Alpha virt. eigenvalues --   -0.92125  -0.92125  -0.47658  -0.01316  -0.01316
 Alpha virt. eigenvalues --   -0.00394   0.04891   0.08529   0.14236   0.14236
 Alpha virt. eigenvalues --    0.61188   0.61188   0.68447   0.68447   0.68777
 Alpha virt. eigenvalues --    1.05682   1.05682   1.53265   1.62902   1.63168
 Alpha virt. eigenvalues --    1.63168   2.92370   2.92370   3.06985   3.20200
 Alpha virt. eigenvalues --    3.20200   4.04108   4.04108   4.10035   4.10035
 Alpha virt. eigenvalues --    4.22655   4.33022   4.37606   4.37606   4.61833
 Alpha virt. eigenvalues --    4.61833   5.14842   5.14842   5.33205   5.51468
 Alpha virt. eigenvalues --    5.51468   5.74343   5.74343   5.77346   5.77346
 Alpha virt. eigenvalues --    5.87937   5.87937   6.13712   6.49111   6.68177
 Alpha virt. eigenvalues --    6.68177   7.11725  11.65391
          Condensed to atoms (all electrons):
               1          2
     1  O    6.538644   0.461356
     2  O    0.461356   6.538644
 Mulliken charges:
               1
     1  O    1.000000
     2  O    1.000000
 Sum of Mulliken charges =   2.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  O    1.000000
     2  O    1.000000
 Electronic spatial extent (au):  <R**2>=             33.1124
 Charge=              2.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0000    Y=              0.0000    Z=              0.0000  Tot=              0.0000
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -6.7387   YY=             -6.7387   ZZ=             -2.7164
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             -1.3408   YY=             -1.3408   ZZ=              2.6815
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              0.0000  YYY=              0.0000  ZZZ=              0.0000  XYY=              0.0000
  XXY=              0.0000  XXZ=              0.0000  XZZ=              0.0000  YZZ=              0.0000
  YYZ=              0.0000  XYZ=              0.0000
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=             -3.5884 YYYY=             -3.5884 ZZZZ=            -16.2770 XXXY=              0.0000
 XXXZ=              0.0000 YYYX=              0.0000 YYYZ=              0.0000 ZZZX=              0.0000
 ZZZY=              0.0000 XXYY=             -1.1961 XXZZ=             -3.4109 YYZZ=             -3.4109
 XXYZ=              0.0000 YYXZ=              0.0000 ZZXY=              0.0000
 N-N= 2.788369819452D+01 E-N=-3.923447127318D+02  KE= 1.470194859562D+02
 Symmetry AG   KE= 7.084563145867D+01
 Symmetry B1G  KE= 3.110475560988D-34
 Symmetry B2G  KE=-2.553667775283D-20
 Symmetry B3G  KE= 1.769231330884D-18
 Symmetry AU   KE= 1.231867805468D-33
 Symmetry B1U  KE= 6.604855798906D+01
 Symmetry B2U  KE= 5.062648254253D+00
 Symmetry B3U  KE= 5.062648254253D+00
 1\1\GINC-GAMBA\SP\RCCSD(T)-FC\Gen\O2(2+)\IRIKURA\17-May-2017\0\\# CCSD
 (T,maxcyc=100)/GEN gfinput geom=check scf=xqc\\BEB step 10: low-spin C
 CSD(T) for dication of o2\\2,1\O,0,-2.024167039,1.50180939,0.\O,0,-3.2
 38759921,1.50180939,0.\\Version=ES64L-G09RevD.01\State=1-SGG\HF=-148.2
 068408\MP2=-148.8269795\MP3=-148.6632502\MP4D=-148.7689281\MP4DQ=-148.
 7525485\MP4SDQ=-148.7772962\CCSD=-148.7353399\CCSD(T)=-148.7718751\RMS
 D=7.304e-09\PG=D*H [C*(O1.O1)]\\@


 IF AT FIRST YOU DON'T SUCCEED, TRY, TRY AGAIN.
 THEN GIVE UP;
 THERE'S NO USE BEING A DAMN FOOL ABOUT IT.

                                 -- W. C. FIELDS
 Job cpu time:       0 days  0 hours  0 minutes 23.6 seconds.
 File lengths (MBytes):  RWF=     26 Int=      0 D2E=      0 Chk=      1 Scr=      1
 Normal termination of Gaussian 09 at Wed May 17 14:07:27 2017.
